dcc-portal-ui/app/scripts/compounds/views/compound.html

```diff
@@ -158,8 +158,11 @@ <h1 data-ui-scrollfix="79">
                         <span data-ng-if="CompoundCtrl.getTargetedCompoundGenes() && CompoundCtrl.getTargetedCompoundGenes().length > 0">
                             <translate has-markup>Showing <strong data-ng-bind="CompoundCtrl.getTargetedCompoundGenesResultPerPage()"></strong> of <strong data-ng-bind="CompoundCtrl.getTargetedGeneCount()"></strong> genes</translate>
                             <span class="t_tools" data-toolbar="" data-dl="compound_mutated_genes">
-                            <span class="t_tools__tool" data-tooltip-placement="left" data-tooltip="{{'Export Table as TSV' | translate}}" data-ng-clicks="downloadHTMLTable(dl, 'tsv')"><i class="icon-file"></i></span>
+                                <span class="t_tools__tool" data-tooltip-placement="left" data-tooltip="{{'Export Table as TSV' | translate}}" data-ng-clicks="downloadHTMLTable(dl, 'tsv')"><i class="icon-file"></i></span>
```

Comment:
Default Indentation space on that particular page is 4. Would you like me to change them to 2?